src/manifests_workflow/component.py

```diff
@@ -28,9 +28,4 @@ def branches(self, url: str) -> List[str]:
         return branches
 
     def to_dict(self) -> Any:
-        return Manifest.compact({
-            "name": self.name,
-            "repository": self.git_repo.url,
-            "ref": self.git_repo.ref,
-            "checks": self.checks
-        })
+        return Manifest.compact({"name": self.name, "repository": self.git_repo.url, "ref": self.git_repo.ref, "checks": self.checks})
```

Comment:
I know the linter undoes this formatting, maybe just revert this file since there are no changes.

Reply:
> I think there's one more problem, the manifests CI workflow is going to pickup these templates. They probably need to move or the folder needs to be excluded somehow.
> 
> https://github.com/opensearch-project/opensearch-build/blob/main/.github/workflows/manifests.yml#L20

This is actually my next step after this PR. I plan to take over this PR to resolve some of the manifest CI failures we have right now. https://github.com/opensearch-project/opensearch-build/pull/2333
I will temporary exclude the templates checks in this PR then.

Reply:
Right, I just don't want this PR to break what works today.